cmds/exp/efivarfs/efivarfs_test.go

```diff
@@ -83,16 +145,58 @@ func TestRun(t *testing.T) {
 				}
 				return s
 			},
-			wantErr: "no efivarfs",
+			wantErr: syscall.EIO,
+		},
+		{
+			name:  "write good variable bad content",
+			e:     okfs,
+			write: " WriteOnceStatus-4b3082a3-80c6-4d7e-9cd0-583917265df1",
+			setup: func(path string, t *testing.T) string {
+				t.Helper()
+				return filepath.Join(path, "content")
+			},
+			wantErr: os.ErrNotExist,
+		},
+		{
+			name:  "write good variable succeeds",
+			e:     okfs,
+			write: " WriteOnceStatus-4b3082a3-80c6-4d7e-9cd0-583917265df1",
+			setup: func(path string, t *testing.T) string {
+				t.Helper()
+				f, err := os.Create(filepath.Join(path, "content"))
+				if err != nil {
+					t.Errorf("Failed to create file: %v", err)
```

Comment:
t.Fatalf?

Reply:
same thing. Errorf is the right answer for this type of test.